lib/containers/__tests__/__snapshots__/gene-tests.js.snap

```diff
@@ -65,7 +65,7 @@ exports[`test looks like expected 1`] = `
         <Text
           accessible={true}
           allowFontScaling={true}
-          lineBreakMode="tail" />
+          ellipsizeMode="tail" />
       </View>
```

Comment:
this is a snapshot test, looks like that's been renamed in React Native 0.32..0.34